README.md

```diff
@@ -120,6 +120,24 @@ workflow dosomething {
 `vidarr_type` and `vidarr_retry` can be combined. Only basic types (Booleans, dates, floats, integers, JSON, and
 strings) may be retried. Input files cannot be changed by retrying.
 
+`vidarr_label` can also be added to `output_meta` to label an output file.
```

Comment:
Please expand a little more on this, in particular that it may upgrade the WDL type to Pair[File,Map[String,String]] so that the vidarr type will be file-with-labels, and that vidarr_type takes priority and will still overwrite the type